test/files/glob-source.spec.js

```diff
@@ -1,14 +1,11 @@
-'use strict'
-
 /* eslint-env mocha */
-const { expect } = require('aegir/utils/chai')
-const globSource = require('../../src/files/glob-source')
-const all = require('it-all')
-const path = require('path')
-const {
-  isNode
-} = require('../../src/env')
-const fs = require('fs')
+import fs from 'fs'
+import path from 'path'
+import { expect } from 'aegir/chai'
+import all from 'it-all'
+import { isNode } from '../../src/env.js'
+import globSource from '../../src/files/glob-source.js'
+const __dirname = import.meta.dirname
```

Comment:
`import.meta.dirname` is [only defined in Node 20+](https://stackoverflow.com/a/50052194) & this package is marked as working in Node 18. For Node 18, it's:

```js
import { dirname } from 'node:path'
import { fileURLToPath } from 'node:url'

const __dirname = dirname(fileURLToPath(import.meta.url))
```

Reply:
`fileURLToPath` is not present in `url` (polyfill for `node:url`), and thus it makes tests fail in Chrome (and other browsers).

Two options:

1) polyfill `fileURLToPath`
2) bump minimum node to 20.x

I prefer the latter because 18 is reaching EOL in 3 months